client/src/contexts/QA.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useContext } from 'react';
+import axios from 'axios';
+
+export const QAContext = React.createContext();
+
+export const QAProvider({ children }) {
+  const fetchAllQuestions = (product_id, page, count, callback) => {
```

Comment:
can rename function to `fetchQuestions`, since you're filtering by the 3 parameters